src/hono-base.ts

```diff
@@ -83,14 +85,65 @@ export type HonoOptions<E extends Env> = {
 
 class Hono<E extends Env = Env, S extends Schema = {}, BasePath extends string = '/'> {
   // Theses methods are dynamically initialized in the constructor.
+  /**
```

Comment:
It seems these docs are not used.

![CleanShot 2024-05-29 at 02 05 40@2x](https://github.com/honojs/hono/assets/10682/510be2bc-382b-4653-806f-72a2bd6bd1e1)

I don't have a good idea to show the docs for the super complex types in the IDE. But, I think these docs for `get!` or `post!` etc. are not necessary.

Reply:
Resolved!